proto/blockchain.proto

```diff
@@ -64,10 +38,13 @@ message VotePb {
 }
 
 message ActionPb {
+    uint64 nonce = 3;
```

Comment:
Can we remove L41-L44 from this PR? Let's keep this PR being a pure Tx and TxPb cleanup change.